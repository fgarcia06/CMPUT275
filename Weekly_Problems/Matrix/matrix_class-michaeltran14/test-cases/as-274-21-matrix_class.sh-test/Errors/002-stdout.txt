g++: error: matrix.o: No such file or directory
g++: fatal error: input file ‘we5_test.o’ is the same as output file
compilation terminated.
make[1]: *** [Makefile:10: we5_solution] Error 1
chmod: cannot access 'we5_solution': No such file or directory
/home/michaeltran14/CMPUT_275/C++/Weekly_Problems/Matrix/matrix_class-michaeltran14/soln/.build/build.sh: line 9: ./we5_solution: No such file or directory

Expected file "test-cases/as-274-21-matrix_class.sh-test/Expected/002-stdout.txt" has no output file match
