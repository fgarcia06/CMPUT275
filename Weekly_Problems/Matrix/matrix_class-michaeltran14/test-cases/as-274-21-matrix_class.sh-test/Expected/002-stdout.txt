-1 -1 -1 -1
-1 -1 -1 -1
-1 -1 -1 -1
2 2 2 2
2 2 2 2
2 2 2 2
3 3 3
3 3 3
3 3 3
3 3 3
-3 -2 -1
0 -3 -2
-1 0 -3
-2 -1 0
0 1 2
3 0 1
2 3 0
1 2 3
1 1 1 1
1 1 1 1
1 1 1 1
3 3 3 3
3 3 3 3
3 3 3 3
-12 -12 -12
-12 -12 -12
-12 -12 -12
-12 -12 -12
-12 -12 -12
-12 -12 -12
-24 -24 -24
-24 -24 -24
-24 -24 -24
-3 -3 -3
-3 -3 -3
-3 -3 -3
-3 -3 -3
-3 0 -1 -2
-2 -3 0 -1
-1 -2 -3 0
-18 -15 -12 -9
-18 -15 -12 -9
-18 -15 -12 -9
-18 -15 -12 -9
3
2 2 2 2
2 2 2 2
2 2 1 2
8 8 8 8
8 8 8 8
8 8 8 8
-1 -1 -1 -1
-1 -1 -1 -1
-1 -1 -1 88
-1 -1 -1 -1
-1 -1 -1 -1
-1 -1 -1 -1
